tests/test_network.py

```diff
@@ -0,0 +1,37 @@
+from autowebcompat import network
+from keras import backend as K
+import math
+import numpy as np
+
+arr1 = np.array([[1, 1]], dtype=np.float32)
+arr2 = np.array([[0, 1]], dtype=np.float32)
+
+arr3 = np.array([[0, 0]], dtype=np.float32)
+arr4 = np.array([[3, 2]], dtype=np.float32)
+
+
+def test_eucledian_distance():
+    dist = network.euclidean_distance([arr1, arr2])
+    dist2 = network.euclidean_distance([arr3, arr4])
+    assert (K.eval(dist) == 1)
+    assert (math.floor(K.eval(dist2) * 100) == 360)
+
+
+def test_eucl_distance_output_shape():
+    vect = [arr1.shape, arr2.shape]
+    assert (network.eucl_dist_output_shape(vect) == (1, 1))
+
+
+def test_contrastive_loss():
+    label = [0, 1]  # possible values for output
+    euclid_dist = network.euclidean_distance([arr1, arr2])  # it returns 1
```

Comment:
You're testing `contrastive_loss` here, so there's no pointing in calling `euclidean_distance` if you already know its result.